### PR TITLE
docs(examples): Add Remix Auth with FormStrategy example

### PR DESCRIPTION
**Pull request by sergiodxa**
This adds an example using [Remix Auth](https://github.com/sergiodxa/remix-auth) with the [FormStrategy](https://github.com/sergiodxa/remix-auth-form).

### CODE REVIEW

File: examples/remix-auth/form/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/remix-auth/form)
+
+## Example
+
+Describe the example and how it demonstrates solving the problem. Reference any relevant files/dependencies if needed.
+
+## Related Links
+
+Link to documentation or other related examples.
```
Comment (sergiodxa) [inline examples/remix-auth/form/README.md:17]: I still have to write this

File: examples/remix-auth/form/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Remix Auth - FormStrategy
+
+This adds an example using [Remix Auth](https://github.com/sergiodxa/remix-auth) with the [FormStrategy](https://github.com/sergiodxa/remix-auth-form) to do authentication with an email and password.
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/remix-auth/form)
```
Comment (kentcdodds) [inline examples/remix-auth/form/README.md:9]: Would it make more sense for the directory to be called "email and password"?

```suggestion
[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/remix-auth/email-and-password)
```

File: examples/remix-auth/form/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## Example
+
+Describe the example and how it demonstrates solving the problem. Reference any relevant files/dependencies if needed.
```
Comment (kentcdodds) [inline examples/remix-auth/form/README.md:13]: TODO :)

**Review by kentcdodds — CHANGES_REQUESTED** [review]
This is awesome! Just those last bits and this will be good to go.

File: examples/remix-auth/form/README.md (hunk 2 above)
Comment (sergiodxa) [reply examples/remix-auth/form/README.md:9]: The strategy is called FormStrategy, you don't necessarily need to use email and password, you could use it to also do username + password, or maybe add a third field.

Comment (kentcdodds) [reply examples/remix-auth/form/README.md:9]: Ah, gotcha 👍

**Review by kentcdodds — APPROVED** [review]
This is great! Thanks 👏

**Pull request merged**